core/focus_manager.ts

```diff
@@ -423,6 +425,8 @@ export class FocusManager {
    * the returned lambda is called. Additionally, only 1 ephemeral focus context
    * can be active at any given time (attempting to activate more than one
    * simultaneously will result in an error being thrown).
+   *
+   * This method does not scroll the ephemerally focused node into view.
```

Comment:
```suggestion
   * This method does not scroll the ephemerally focused element into view.
```

Nit on terminology for consistence.

Reply:
Done.